microapps/ArWorldScaleApp/app/src/main/java/com/arcgismaps/toolkit/arworldscaleapp/screens/MainScreen.kt

```diff
@@ -204,7 +226,7 @@ fun MainScreen() {
                     modifier = Modifier
                         .padding(it)
                         .fillMaxSize(),
-                    worldScaleTrackingMode = trackingMode,
+                    worldScaleTrackingMode = selectedTrackingMode,
```

Comment:
missing clipping distance:
```suggestion
                    worldScaleTrackingMode = selectedTrackingMode,
                    clippingDistance = 100.0,
```